### ISSUE TITLE
Crashing from spells / particles permanently disappearing

### ISSUE THREAD

**Issue by FieryAlex** [pale-imitations/Magic_Kingdoms#11]
This is already one of if not my favorite mod but it does have 2 issues when i play

Issue 1: it will occasionally crash when i use a spell and sends everyone in a server back around 10 irl minutes. I tested it on a modpack with only magic kingdoms installed and it still crashed (the banishment spell crashes really early on).

Issue 2: After playing on a world for a couple of days the particle effects for all the spells just disappeared and i never saw them again, i tried disabling particle effects and enabling them again but still no particle effects. its only the magic kingdoms particle effects that disappear and its only in that world.

I love this mod and am looking forward to the updates and everything in the future for this mod and its already a really good mod even though it came out 1-2 months ago (that is what i heard so sorry if it came out later)

**Comment by FieryAlex** [comment]
Update: after going back into the world after it crashed again the particle effects are back (they might go away again in the future but they are back for now) but it does still crash after using some spells.

**Comment by pale-imitations** [comment]
Could you post the crash reports, it'll help me diagnose the issue

**Comment by FieryAlex** [comment]
> Could you post the crash reports, it'll help me diagnose the issue

Yeah sure, minecraft decided to stop creating crash reports so i am just going to link all the crash reports i had so if one is not magic kingdoms sorry about that but at-least 2 should be magic kingdoms and i wasn't really sure how to tell which were magic kingdoms.

[crash-2019-12-30_13.23.35-client.txt](https://github.com/pale-imitations/Magic_Kingdoms/files/4022183/crash-2019-12-30_13.23.35-client.txt)

[crash-2019-12-29_23.18.16-client.txt](https://github.com/pale-imitations/Magic_Kingdoms/files/4022186/crash-2019-12-29_23.18.16-client.txt)

[crash-2019-12-29_13.59.16-client.txt](https://github.com/pale-imitations/Magic_Kingdoms/files/4022187/crash-2019-12-29_13.59.16-client.txt)

**Comment by pale-imitations** [comment]
Unfortunately, none of these crash reports are pinging any lines of MK's code. So I have no idea where the issue is. Is this still an issue for you or did any of the recent updates fix it?

**Comment by FieryAlex** [comment]
I just went into the world with an updated version and it seems to be
fixed. I went on a spree using the banish spell and i ran out of mana with
no crashing (usually it crashed on the first or second use) and i haven't
been playing minecraft a lot recently so when i saw the updated version i
was kinda blown away by how fast progress on this mod is going. I look
forward to seeing the future of already my favorite mod.
